More information about payg clients (#2336)
uyuni-project/uyuni-docs: modules/installation-and-upgrade/pages/connect-payg.adoc

* operates per-region RMT Servers mirroring repositories for products available as {payg}

This document describes how to connect existing {payg} instance to {productname} server, and gives basic information about credentials collection from the instance.

> The goal of the connection is to extract authentication data so {productname} server is able to connect to cloud RMT host. That will give the {productname} server access to products that are on the RMT host and that are not already available via the {scc} organization credentials.

Reviewer: The goal of this connection is to extract authentication data so the {productname} server can connect to a cloud RMT host. {productname} server then has access to products on the RMT host which are not available via {scc} organization credentials.

Author: Yes, please remind I'm not a native speaker.

But maybe keep "the" in front of "{scc} organization credentials" ? we are speaking of specific credentials, the ones that are already entered ; not any credentials in general.

> The goal of the connection is to extract authentication data so {productname} server is able to connect to cloud RMT host. That will give the {productname} server access to products that are on the RMT host and that are not already available via the {scc} organization credentials.

Reviewer 2: Here is my suggestion based on both your proposals:
```suggestion
The goal of this connection is to extract authentication data so the {productname} Server can connect to a cloud RMT host.
{productname} Server then has access to products on the RMT host which are not already available with one of the {scc} organization credentials.
```

Before using {payg} feature make sure that: 

* the {payg} instance is launched from the correct SUSE product image (e.g. SLES, SLES for SAP, SLE HPC) to allow access to the desired repositories
* {productname} server has connectivity to the {payg} instance (ideally in the same region) either directly or via a bastion

> * valid {scc} credentials are entered in menu:Admin[Setup Wizard > Organization Credentials], even they provide only limited access to products - your subscription should at least offer access to the {productname} client tools.

Reviewer: * Enter valid {scc} credentials under menu:Admin[Setup Wizard > Organization Credentials], your subscription offers access to the {productname} client tools.

Author: No. We are speaking of a minimum content of the credentials. The client tools are a requisite so it works.

> * valid {scc} credentials are entered in menu:Admin[Setup Wizard > Organization Credentials], even they provide only limited access to products - your subscription should at least offer access to the {productname} client tools.

Reviewer 2: ```suggestion
* valid {scc} credentials are entered in menu:Admin[Setup Wizard > Organization Credentials], even they provide only limited access to products.
  Your subscription should at least offer access to the {productname} client tools.
```

* operates per-region RMT Servers mirroring repositories for products available as {payg}

This document describes how to connect existing {payg} instance to {productname} server, and gives basic information about credentials collection from the instance.
The goal of this connection is to extract authentication data so the {productname} Server can connect to a cloud RMT host. 

> This gives the {productname} server access to products that are on the RMT host that are not already available via the {scc} organization credentials.

Reviewer 2: ```suggestion
Then the {productname} Server has access to products on the RMT host that are not already available with the {scc} organization credentials.
```

Before using {payg} feature make sure that: 

> * The {payg} instance is launched from the correct SUSE product image (e.g. SLES, SLES for SAP, SLE HPC) to allow access to the desired repositories
> 
> * {productname} server has connectivity to the {payg} instance (ideally in the same region) either directly or via a bastion

Reviewer 2: ```suggestion
* The {payg} instance is launched from the correct SUSE product image (for example, SLES, SLES for SAP, or SLE HPC) to allow access to the desired repositories

* {productname} Server has connectivity to the {payg} instance (ideally in the same region) either directly or via a bastion
```

To monitor the syncing progress of each channel, check the log files in the [path]``/var/log/rhn/reposync`` directory on the {productname} Server.

[IMPORTANT]
====

> If a product is provided by both the {payg} instance and by the {scc} subscription, it will appear only once in the products list.

Reviewer: If a product is provided by both the {payg} instance and by an {scc} subscription, it will appear only once in the products list.

> If a product is provided by both the {payg} instance and by the {scc} subscription, it will appear only once in the products list.

Reviewer 2: ```suggestion
If a product is provided by both the {payg} instance and one of the {scc} subscriptions, it will appear only once in the products list.
```
